### DISCUSSION TITLE
Assign Colors to Data

### DISCUSSION THREAD

**Discussion by cameronmcefee** [TelemetryDeck/Viewer#345] [Ideas]
I'm not sure this would get enough use to warrant the time to build it, but I figure I'll throw it out there. I track data that has a strong association with color — I make plugins that work in different design software, each have their own distinct brand colors. Photoshop is blue, Illustrator is yellow, etc. I have a few charts where I split data by host. They somehow magically actually line up kind of close (Photoshop is blue, Illustrator is yellow, surprisingly), but it'd be nice if I could define colors for certain criteria so that I could know at a glance what each line means.

![Screenshot 2024-01-31 at 9 06 51 AM](https://github.com/TelemetryDeck/Issues/assets/72919/683e74e3-012a-4ae6-9c0e-4dc45c0df249)

**Comment by winsmith** [comment]
Love it, but unsure how that would work technically. You'd have to provide a mapping somewhere, maybe in the query object?

**Reply by cameronmcefee** [reply]
Yeah, that seems like the most practical place to put it, since it's probably a very niche use that only people with very specific needs will want.

Alternatively, I notice that the settings for a project have a color scheme option. I could see putting some kind of GUI interface there where you define colors and matching values based on your existing data.
